lib/house.rb

```diff
@@ -0,0 +1,37 @@
+class House
+  attr_reader :price, :address, :rooms
+  def initialize(price, address)
+    @price = price[1..-1].to_i
+    @address = address
+    @rooms = []
+  end
+
+  def add_room(room)
+    @rooms << room
+  end
+
+  def above_market_average?
+    @price > 500000
+  end
+
+  def rooms_from_category(category)
+    @rooms.find_all do |room|
+      room.category == category
+    end
+  end
+
+  def area
```

Comment:
Good job using `each` to make this calculation - you could also use sum like this:

```ruby
@rooms.sum do |room|
  room.area
end
```